test/index.js

```diff
@@ -63,16 +62,17 @@ test('handles aliased props', t => {
   })
 })
 
-test('long form prop trumps aliased props', t => {
-  const style = backgroundColor({
-    theme,
-    backgroundColor: 'black',
-    bg: 'blue',
-  })
-  t.deepEqual(style, {
-    backgroundColor: '#111',
-  })
-})
+// Impossible to ensure, due to perf issues
```

Comment:
Does this mean that the last prop defined now trumps (i.e. object order)?

Reply:
Yeah exactly, the last property defined on the object will won. It is very predictable and easy to fix.